playback/tape_cassettes/in_memory/in_memory_tape_cassette.py

```diff
@@ -1,3 +1,4 @@
+# pylint: disable=duplicate-code
```

Comment:
after increasing the minimum, is this still necessary?

Reply:
Weird I thought I removed this